src/ludamus/templates/crowd/user/party_detail.html

```
@@ -0,0 +1,287 @@
+                                <button type="button"
+                                        class="icon-btn relative"
+                                        data-copy="{{ claim_path }}"
+                                        data-copy-origin
+                                        data-copied-label="{% translate "Copied!" %}"
+                                        title="{% translate "Copy claim link" %}">
+                                    {% icon "clipboard-document" class="w-3.5 h-3.5" %}
+                                    <span class="sr-only">{% translate "Copy claim link" %}</span>
+                                    {% tessera_copy_popover %}
+                                </button>
+                            {% endif %}
+                            <form method="post"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)